security.sh

```diff
@@ -1,7 +1,9 @@
 #!/usr/bin/env bash
 echo ${TEST_URL}
-zap-api-scan.py -t ${TEST_URL}/v2/api-docs -f openapi -u ${Rules} -P 1001
+export LC_ALL=C.UTF-8
+export LANG=C.UTF-8
+zap-api-scan.py -t ${TEST_URL}/v2/api-docs -f openapi -S -d -u ${SecurityRules} -P 1001
 cat zap.out
-zap-cli --zap-url http://0.0.0.0 -p 1001 report -o /zap/api-report.html -f html
-cp /zap/api-report.html functional-output/
-zap-cli -p 1001 alerts -l Informational
\ No newline at end of file
+curl --fail http://0.0.0.0:1001/OTHER/core/other/jsonreport/?formMethod=GET --output report.json
+cp *.* functional-output/
+zap-cli --zap-url http://0.0.0.0 -p 1001 alerts -l Medium --exit-code False
```

Comment:
Same as above, bought in from master, not changed in this PR. Caused by develop being a bit behind